### DISCUSSION TITLE
Sharing some tables between two Directus environments

### DISCUSSION THREAD

**Discussion by kamil-malinski** [directus/directus#18002] [Archive]
Hi!

I have two very similar Directus environments (.site1.env and .site2.env). These two have the same codebase, extensions etc. but use two different databases (db1 and db2 on the same mysql-server). I would like to share some tables between those two databases (e.g. the directus_users-table).

I tried to create views in db1 from certain tables in db2 but had no success, because Directus seems to need real tables and not views. The second best idea was to sync the tables with triggers, but it seems to be very troublesome to create triggers for all the tables, events (update, insert, delete etc.). Is there a better way to do this?

Kamil

**Comment by rijkvanzanten** [comment]
There's no real way to do that right now. Ideally, Directus would be able to connect to 3 databases at the same time (shared + env1 + env2), but that's not currently the case yet 👍🏻
